strings/base_error.h

```diff
@@ -584,7 +584,7 @@ namespace winrt::impl
 {
     inline hresult check_hresult_allow_bounds(hresult const result)
     {
-        if (result != impl::error_out_of_bounds)
+        if (result != impl::error_out_of_bounds && result != impl::error_fail && result != impl::error_file_not_found)
```

Comment:
Wish there was some way to incentivize the implementations to clean up their act as this isn't sustainable and they're not likely to fix this on their end.

Reply:
It's a contract breaking change now if we "fix" the XAML implementation.